apps/services/endorsements/api/src/app/modules/endorsement/endorsement.service.ts

```diff
@@ -445,6 +445,9 @@ export class EndorsementService {
         },
       ],
     })
+    this.logger.debug(
```

Comment:
can/should this be higher? doesn't sound to me like this will be a high volume of logs. consider info.

Reply:
good call ... info it is